ui/apps/platform/cypress/integration/vulnerabilities/workloadCves/workloadCveOverviewPage.test.ts

```diff
@@ -123,7 +123,7 @@ describe('Workload CVE overview page tests', () => {
     });
 
     describe('SBOM generation tests', () => {
-        const sbomGenerationButtonSelector = 'button:contains("Generate SBOM")';
+        const sbomGenerationButton = 'button:contains("Generate SBOM")';
```

Comment:
While noticing that these tests are parallel to the preceding file (bravo) one thought came to mind:
If **Generate SBOM** is in row actions menu here, what is pro and con to make that explicit in the selector?

Reply:
Maybe to look at it another way, how about separate variables for the 2 selectors?
In case the text in the row action and in the modal ever became different, like **Confirm**.
Ditto above for button on page versus on modal.

Reply:
I like this idea, and it makes the separation between the row item and header button for the two pages more distinct as well.